jme3-terrain/src/main/resources/Common/MatDefs/Terrain/HeightBasedTerrain.j3md

```diff
@@ -42,4 +42,4 @@ MaterialDef Terrain {
 
     Technique {
     }
-}
\ No newline at end of file
+}

```

Comment:
You can also restore this file, as it doesn’t contribute to the functionality of your PR